clarifai/client/input.py

```diff
@@ -761,6 +811,65 @@ def upload_annotations(self, batch_annot: List[resources_pb2.Annotation], show_l
 
     return retry_upload
 
+  def patch_annotations(self, batch_annot: List[resources_pb2.Annotation],
+                        action: str = 'merge') -> None:
+    """Upload image annotations to app.
```

Comment:
Description of the function is wrong here.

Reply:
Done